gamechoice.rb

```diff
@@ -0,0 +1,7 @@
+class GameChoice
+  OPTIONS = [:Rock, :Paper, :Scissors]
```

Comment:
Great use of a constant here, makes the function of the options array really clear!